openapi-specifications/test-data-generator.json

```diff
@@ -0,0 +1,196 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Test Data Generator",
+    "description": "Generate company test data",
+    "version": "1.0.0"
+  },
+  "servers": [
+    {
+      "url": "https://api.companieshouse.gov.uk/test-data"
+    }
+  ],
+  "tags": [
+    {
+      "name": "company",
+      "description": "Create a company for testing"
+    }
+  ],
+  "paths": {
+    "/company": {
+      "post": {
+        "tags": [
+          "company"
+        ],
+        "summary": "Create a new test company",
+        "operationId": "createCompany",
+        "requestBody": {
+          "description": "Company Spec to customise your requested company. If not supplied default values will be used",
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/CompanySpec"
+              }
+            }
+          },
+          "required": false
+        },
+        "responses": {
+          "201": {
+            "description": "Successful creation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/CreatedCompany"
+                }
+              }
+            }
+          },
+          "400": {
```

Comment:
When would you get a general "bad request" that doesn't have a more specific. http response?

Reply:
Bad request will be returned when the request body is either a malformed json or contains an invalid value